tests/integration_tests/canal_json_basic/run.sh

```diff
@@ -27,6 +27,9 @@ function run() {
 	SINK_URI="kafka://127.0.0.1:9092/test?protocol=canal-json&kafka-version=${KAFKA_VERSION}&max-message-bytes=10485760"
 
 	run_cdc_cli changefeed create --start-ts=$start_ts --sink-uri="$SINK_URI"
+	if [ "$SINK_TYPE" == "kafka" ]; then
```

Comment:
Please add a new test, which now relies on canal-adapter for synchronization.

Reply:
it looks that since the topic name is `test`, then be consumed by `canal-adapter`.

This case is used to check that TiCD Canal-JSON encoder's output is compatible with `canal-json`, I will rename the case to `canal_json_adapter_compatibility`, and then add a new `canal_json_basic` case.